config/set/coding-style/coding-style.php

```diff
@@ -99,5 +100,7 @@
 
     $services->set(UnderscoreToPascalCaseVariableNameRector::class);
 
+    $services->set(VariableVariablesRector::class);
```

Comment:
Btw, this name is not very clear IMO. Could you think of better name that described what is being changed from to?

Reply:
`WrapVariableVariables`? `WrapVariableVariablesInBraces`?

Unfortunately "variable variables" is exactly what this Rector applies to.

Open to other suggestions...

Reply:
`UnwrappedVariableVariableNameToCurlyBracesVariableVariableNameRector`?

Reply:
The first word should be a verb, since Rector is about changing from to.

But the rest is good :+1:

Reply:
In that case it should be using the word "wrap": `WrapVariableVariableNameInCurlyBracesRector`